XRTK.SDK/Packages/com.xrtk.sdk/Runtime/Features/Diagnostics/Scripts/MixedRealityFrameDiagnosticsHandler.cs

```diff
@@ -33,14 +35,33 @@ public class MixedRealityFrameDiagnosticsHandler : MonoBehaviour,
         [Tooltip("The text component used to display the GPU FPS information.")]
         private TextMeshProUGUI gpuFrameRateText = null;
 
+        [Prefab]
         [SerializeField]
-        [Tooltip("Image components used to visualize missed frames.")]
-        private Image[] missedFrameImages = null;
+        [Tooltip("Image component used to visualize missed frames.")]
```

Comment:
These changes seem unrelated to this PR

Reply:
I mean I guess I could pull it out but originally I was trying to fix that prefab anyway and did lol.

Reply:
I am fine with not pulling it out I guess. It's gonna get in there anyway.